### PR TITLE
Add timeperiod CLI support

### LINKED ISSUE #245: Add Timeperiod support to CLI
For allowing better programmatic use of the Engine, specifying the timeperiod via CLI should be possible

Flag could be `--timeperiod` for example.

Format could be 20200101-20200201

### PR DESCRIPTION
**Pull request by marijn111**
# Results of merging this

Fixes #245 

# What has been changed?

two flags have been added to the CLI arguments: `-from` and `-to`, corresponding respectively to `backtesting-from` and `backtesting-to`.

# Examples

# Actions to be performed before this can be merged

# Security Measures

N/A

# Potential Issues / Future considerations

N/A

# Assumptions made / Things to add to the wiki
 
Changed the type from datetime to string, since the value entered is a string (right?). Only later gets the string transformed into a datetime object. 

Furthermore, no validations have been written, since the CLI basically overwrites the config.json. Afterwards, the config.json gets validated either way. 

# Packages added

N/A

# Additional Info

### CODE REVIEW

**Review by geordle — APPROVED** [review]

File: modules/setup/config/specification.json [hunk 1]
```diff
@@ -26,11 +26,17 @@
   },
   {
     "name": "backtesting-from",
-    "type": "datetime"
+    "type": "string",
+    "cli": {
+      "short": "start"
```
Comment (geordle) [inline modules/setup/config/specification.json:31]: IMO `start` => `from`  and `end` => `to` would be more in line with preexisting 
vars `backtesting-from` and `backtesting-to`. Might build on existing knowledge. Good enough though

**Review by geordle — APPROVED** [review]

File: modules/setup/config/specification.json (hunk 1 above)
Comment (geordle) [reply modules/setup/config/specification.json:31]: Resolve as you see fit

Comment (keebrev) [reply modules/setup/config/specification.json:31]: @geordle would agree on this one.

**Comment by marijn111** [comment]
@geordle I too agree that thats better, thanks. Modified accordingly

**Review by geordle — APPROVED** [review]

**Review by sheridavandenbent — APPROVED** [review]

**Comment by keebrev** [comment]
@marijn111 I've updated this branche with `development`. PR can be merged now :)

**Pull request merged**